include/zephyr/pm/policy.h

```diff
@@ -195,21 +195,21 @@ void pm_policy_latency_changed_unsubscribe(struct pm_policy_latency_subscription
  * *far* future, that is, after the cycle counter rollover.
  *
  * @param evt Event.
- * @param time_us When the event will occur, in microseconds from now.
+ * @param cycle When the event will occur, in absolute time (cycles).
  *
  * @see pm_policy_event_unregister
  */
-void pm_policy_event_register(struct pm_policy_event *evt, uint32_t time_us);
+void pm_policy_event_register(struct pm_policy_event *evt, uint32_t cycle);
```

Comment:
Please, add information in the docstring above that the CPU is woken up before the time passed in `cycle` to prevent the event handling latency.